### PR TITLE
Change tag linux-login-casp to linux-login-microos

### PR DESCRIPTION
**Pull request by jlausuch**

### CODE REVIEW

File: linux-login-kubic-20180201.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   ],
   "tags": [
     "ENV-DISTRI-KUBIC",
```
Comment (asmorodskyi) [inline linux-login-kubic-20180201.json:26]: what about this tag ? does it still make sense ?

**Comment by Vogtinator** [comment]
The test PR was merged already...

**Comment by asmorodskyi** [comment]
> The test PR was merged already...

was about to merge this one too , but your were faster :)

File: linux-login-kubic-20180201.json (hunk 1 above)
Comment (jlausuch) [reply linux-login-kubic-20180201.json:26]: I guess so, there are flavors `Kubic-DVD` and `MicroOS-Image-Kubic-kubeadm` in O3

**Pull request merged**